src/data-loader.ts

```
@@ -567,7 +567,13 @@ export function formatDate(dateStr: string, timezone?: string, locale?: string):
  * @returns Formatted date string with newline separator (YYYY\nMM-DD)
  */
 export function formatDateCompact(dateStr: string, timezone: string | undefined, locale: string): string {
-	const date = new Date(dateStr);
+	// For YYYY-MM-DD format, append T00:00:00 to parse as local date
+	// Without this, new Date('YYYY-MM-DD') interprets as UTC midnight
+	const date = dailyDateSchema.safeParse(dateStr).success
```

ESLint

[error] Unexpected any value in conditional. An explicit comparison or type conversion is required.

(ts/strict-boolean-expressions)